diff --git a/ElvUI/Core/General/Toolkit.lua b/ElvUI/Core/General/Toolkit.lua
index f32c98da2..023f92dfc 100644
--- a/ElvUI/Core/General/Toolkit.lua
+++ b/ElvUI/Core/General/Toolkit.lua
@@ -39,7 +39,7 @@ local function DisablePixelSnap(frame)
 			frame:SetTexelSnappingBias(0)
 		elseif frame.GetStatusBarTexture then
 			local texture = frame:GetStatusBarTexture()
-			if texture and texture.SetSnapToPixelGrid then
+			if texture and type(texture) ~= "number" and texture.SetSnapToPixelGrid then
 				texture:SetSnapToPixelGrid(false)
 				texture:SetTexelSnappingBias(0)
 			end
diff --git a/ElvUI/Core/Modules/Bags/Bags.lua b/ElvUI/Core/Modules/Bags/Bags.lua
index b36d50990..7258a039f 100644
--- a/ElvUI/Core/Modules/Bags/Bags.lua
+++ b/ElvUI/Core/Modules/Bags/Bags.lua
@@ -1337,7 +1337,12 @@ function B:GetGrays(vendor)
 			if itemLink and not info.hasNoValue and not B.ExcludeGrays[info.itemID] then
 				local _, _, rarity, _, _, _, _, _, _, _, itemPrice, classID, _, bindType = GetItemInfo(itemLink)
 
+				local isTransmogUncollected = CanIMogIt
+					and CanIMogIt:IsEquippable(itemLink)
+					and not CanIMogIt:PlayerKnowsTransmog(itemLink);
+
 				if rarity and rarity == 0 -- grays :o
+				and not isTransmogUncollected
 				and (classID ~= 12 or bindType ~= 4) -- Quest can be classID:12 or bindType:4
 				and (not E.Retail or not IsCosmeticItem(itemLink) or C_TransmogCollection_PlayerHasTransmogByItemInfo(itemLink)) then -- skip transmogable items
 					local stackCount = info.stackCount or 1
diff --git a/ElvUI/Core/Modules/Chat/Chat.lua b/ElvUI/Core/Modules/Chat/Chat.lua
index 8cae89ee1..ac38f0f3d 100644
--- a/ElvUI/Core/Modules/Chat/Chat.lua
+++ b/ElvUI/Core/Modules/Chat/Chat.lua
@@ -124,6 +124,7 @@ local hyperlinkTypes = {
 	instancelock = true,
 	item = true,
 	keystone = true,
+	mawpower = true,
 	quest = true,
 	spell = true,
 	talent = true,
@@ -1470,11 +1471,11 @@ local function HyperLinkedCPL(data)
 			local visibleLine = chat.visibleLines and chat.visibleLines[lineIndex]
 			local message = visibleLine and visibleLine.messageInfo and visibleLine.messageInfo.message
 			if message and not CH:MessageIsProtected(message) then
-				message = gsub(message,'|c(%x-)|H(.-)|h(.-)|h|r','\10c%1\10H%2\10h%3\10h\10r') -- strip colors and trim but not hyperlinks
-				message = gsub(message,'||','\11') -- for printing item lines from /dump, etc
+				message = gsub(message,'|c(%x-)|H(.-)|h(.-)|h|r','\17c%1\17H%2\17h%3\17h\17r') -- strip colors and trim but not hyperlinks
+				message = gsub(message,'||','\18') -- for printing item lines from /dump, etc
 				message = E:StripString(removeIconFromLine(message))
-				message = gsub(message,'\11','||')
-				message = gsub(message,'\10c(%x-)\10H(.-)\10h(.-)\10h\10r','|c%1|H%2|h%3|h|r')
+				message = gsub(message,'\18','||')
+				message = gsub(message,'\17c(%x-)\17H(.-)\17h(.-)\17h\17r','|c%1|H%2|h%3|h|r')
 
 				if message ~= '' then
 					CH:SetChatEditBoxMessage(message)
diff --git a/ElvUI_Libraries/Core/oUF/elements/tags.lua b/ElvUI_Libraries/Core/oUF/elements/tags.lua
index 6770e2991..630eed969 100644
--- a/ElvUI_Libraries/Core/oUF/elements/tags.lua
+++ b/ElvUI_Libraries/Core/oUF/elements/tags.lua
@@ -806,7 +806,7 @@ local function Tag(self, fs, tagstr, ...)
 		fs.__HookedAlphaFix = true
 	end
 
-	tagstr = tagstr:gsub('||([TCRAtcra])', escapeSequence)
+	tagstr = tagstr:gsub('||([TCRAtcran])', escapeSequence)
 
 	local customArgs = tagstr:match('{(.-)}%]')
 	if customArgs then
